src/common.c

```diff
@@ -70,6 +70,20 @@ void replyRedirect(RedisModuleCtx *ctx, int slot, NodeAddr *addr)
     RedisModule_ReplyWithError(ctx, buf);
 }
 
+/* Create a -ASK reply. */
+void replyAsk(RedisRaftCtx *rr, RedisModuleCtx *ctx, int slot)
+{
+    ShardGroup *sg = rr->sharding_info->importing_slots_map[slot];
+    if (!sg) {
+        RedisModule_ReplyWithError(ctx, "ERR no importing shard group to ask");
+        return;
+    }
+
+    char buf[sizeof(sg->nodes[0].addr.host) + 256];
+    snprintf(buf, sizeof(buf), "ASK %d %s:%u", slot, sg->nodes[0].addr.host, sg->nodes[0].addr.port);
```

Comment:
do we want to iterate through nodes?

Reply:
@yossigo thoughts?  inside a cluster we had a mechanism for iterating through the nodes of the cluster.  I'm wondering if its as useful here.  a cluster client should have a picture of the topology anyways....

Reply:
This issue was raised in different comments below.